Batch File/Batch File (Compound).sublime-syntax

```diff
@@ -21,4 +21,4 @@ hidden: true
 extends: Packages/Batch File/Batch File.sublime-syntax
 
 variables:
-  command_terminator_chars: '[|&<>)]'
+  eoc_char: '[\n|&)]'
```

Comment:
If I correctly understand the ramifications of adding a newline here, this:
```batchfile
for /f "usebackq delims=" %%i in (`echo a
echo b`) echo %%~i
```
Would make you think that the output will be `a` followed by `b`. But it actually is:
```batchfile
for /f "usebackq delims=" %%i in (`echo a echo b`) echo %%~i
```
And echoes `a echo b`.

Newlines don't terminate embedded commands, but are folded to a single space (like YAML's folded scalars).

Reply:
The original intent was to simplify 

https://github.com/sublimehq/Packages/blob/34945b38165ee884ae5a0047e2e279692f0050f6/Batch%20File/Batch%20File.sublime-syntax#L46

by moving `$` to `eoc_char`, but well, this may have unwanted side effects.

Reply:
Each command ends at eol, even in compund commands. The correct example for why `\n` is there would be:

```dosbatch
for %%i in (a b c) do (
	echo a
	echo b
)
```

Without `\n` in `eoc_char` it would look like:

![grafik](https://github.com/sublimehq/Packages/assets/16542113/951d1514-20ff-40ec-bfac-6b619c3c899f)


But the 2nd `echo` is an individual command:

![grafik](https://github.com/sublimehq/Packages/assets/16542113/1657356f-c65f-4ee9-990f-cea82f4efcd3)

Reply:
My bad, I misunderstood what you mean by that. Correct, these are individual commands.